docs/language-guide.rst

```diff
@@ -269,14 +269,13 @@ Illegal symbols are defined as a a single regular expression:
 Pre-defined modes and regular expressions
 -----------------------------------------
 
-Many languages share common modes and regular expressions. Such expressions are defined in core highlight.js code
-at the end under "Common regexps" and "Common modes" titles. Use them when possible.
+Many languages share common modes and regular expressions. Such expressions are defined in `lib/modes.js`. Once compiled, they be found near the end of the core `highlight.js` code under "Common regexps" and "Common modes" titles. Use them when possible.
 
 
 Regular Expression Features
 ---------------------------
 
-The goal of Highlight.js is to support whatever regex features Javascript itself supports.  You're using real regular expressions, use them responsibly.  That said, due to the design of the parser, there are some caveats.  These are addressed below.
+The goal of highlight.js is to support whatever regex features JavaScript itself supports.  You're using real regular expressions, use them responsibly.  That said, due to the design of the parser, there are some caveats.  These are addressed below.
```

Comment:
```suggestion
The goal of Highlight.js is to support whatever regex features JavaScript itself supports.  You're using real regular expressions, use them responsibly.  That said, due to the design of the parser, there are some caveats.  These are addressed below.
```

Our official name is indeed capitalized.  If the reference is to the library or npm module specifically then sometimes it should be lowercase, but not here.

Reply:
If you're going to fix "JavaScript" can you please do it in all the documentation `rst` files... I see 10 or so mentions of it with the incorrect casing.